### ISSUE TITLE
Click and wide attack controls should be swapped

### ISSUE THREAD

**Issue by metalgearsloth** [space-wizards/space-station-14#20401]
Probably makes more sense than the current way.

**Comment by MiraslauKavaliou** [comment]
I disagree, ~~since the left click attack is more of like a critical attack and it makes sense to be the main one.~~ Main reasoning is just people being used to the narrower attack being left click. I think a good compromise would be adding an options > controls toggle to switch between them and then we can make the new switched one the default so new players get used to it but older players get to keep their habit. Definetly should have more discussion on this

**Comment by metalgearsloth** [comment]
> Main reasoning is just people being used to the narrower attack being left click.

game's still playtest so whether people are used to it or not shouldn't have much bearing.

**Comment by KingOnBudget** [comment]
What does this change achive other then disorienting players. And what about mobs that can't wide swing? Will they have to right click to do their normal attack or are they gonna be kept as they are?

**Comment by MiraslauKavaliou** [comment]
I agree with that, the implementation of something like that is confusing. It is customary that the primary action is left-click, it would be really weird to have left-click do nothing and right-click to attack.

**Comment by PixelTheKermit** [comment]
This is likely going to mess up shoving.

**Comment by UbaserB** [comment]
I dont really want a change like this to be implemented either. It simply feels like it doesnt match up correctly, not just because of what we’re used to but also from a gameplay stance.

**Comment by VasilisThePikachu** [comment]
Don't we have keybind options? Just swap your keys around

**Comment by deltanedas** [comment]
this would also be horrible for crusher

**Comment by keronshb** [comment]
> > Main reasoning is just people being used to the narrower attack being left click.
> 
> game's still playtest so whether people are used to it or not shouldn't have much bearing.

It definitely goes past the playtest argument.

In SS13, left click is attack, right click is an extra action or alternative attack/fire.

Using SS13 as an arguing point can have its place and l also don't think it's the end all be all, but it's also a good starting point.

Since we do have rebindable keys, I think we should more focus on getting an official menu with saveable bindable keys since this falls into more "do you put your sword on B or A"

And to be fair, most default controls for games have left click as attack (whether it's shoot or melee).
